packages/main/src/TabContainer.js

```diff
@@ -371,13 +376,14 @@ class TabContainer extends UI5Element {
 
 	onBeforeRendering() {
 		// update selected tab
-		const tabs = this._getTabs();
-		if (tabs.length) {
-			const selectedTabs = tabs.filter(tab => tab.selected);
+		this._allItemsAndSubItems = [];
+		this._getAllSubItems(this._getTabs(), this._allItemsAndSubItems, 1);
```

Comment:
refactor _getAllSubItems so it returns a new array instead of mutating its arguments